vagrant/base.sh

```diff
@@ -17,23 +17,33 @@
 
 set -e
 
-if [ -z `which javac` ]; then
-    apt-get -y update
-    apt-get install -y software-properties-common python-software-properties
-    add-apt-repository -y ppa:webupd8team/java
-    apt-get -y update
-
-    # Try to share cache. See Vagrantfile for details
-    mkdir -p /var/cache/oracle-jdk7-installer
-    if [ -e "/tmp/oracle-jdk7-installer-cache/" ]; then
-        find /tmp/oracle-jdk7-installer-cache/ -not -empty -exec cp '{}' /var/cache/oracle-jdk7-installer/ \;
+if [ -e /etc/redhat-release ]
```

Comment:
This is just my ignorance -- is this a reliable way to detect any RH/Fedora-based release, e.g. no major derivative distros use different naming?